Healthy/Classes/SceneDelegate.swift

```diff
@@ -15,8 +15,8 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         // (see `application:configurationForConnectingSceneSession` instead).
         guard let windowScene = (scene as? UIWindowScene) else { return }
 
-        let viewController = UIViewController()
-        viewController.view.backgroundColor = .systemPink
+        let viewModel = SplashViewModel()
+        let viewController = SplashViewController(viewModel: viewModel)
```

Comment:
I think using `splashViewModel` and `splashViecontroller` as variable name would be more declaritve.

Reply:
✅